lib/server/routes/users.js

```diff
@@ -286,6 +286,24 @@ UsersRouter.prototype.createPasswordResetToken = function(req, res, next) {
 UsersRouter.prototype.confirmPasswordReset = function(req, res, next) {
   const User = this.storage.models.User;
 
+  /*
+   * Mitigate timing attack
+   *  attempt to make non-lookup-responses take similar
+   *  amounts of time as lookup-responses
+   */
+  const count = User.count();
```

Comment:
I believe this returns a cursor, so you need to pass a query object and callback to get the count. Check the mongoose docs.